os/board/rtl8721csm/Kconfig

```diff
@@ -48,5 +48,12 @@ config AMEBAD_FLASH_BLOCK_SIZE
         Setting this vaule to configure erase unit size of esp32 SPI flash.
 	
 endmenu
+
+config USER_SIGN_PREPEND_SIZE
```

Comment:
This is TizenRT signature config, not board config. Please move this near signing config.